readme/memcache.md

```diff
@@ -16,184 +16,183 @@ BLT treats Memcache module integration as opt-in as it can be used only in envir
  *
  * Autoload memcache classes and service container in case module is not installed. 
  * Avoids the need to patch core and allows for overriding the default backend
- * when installing drupal: See https://www.drupal.org/node/2766509.
+ * when installing Drupal: See https://www.drupal.org/node/2766509.
  */
 
 if ($is_ah_env) {
   switch ($ah_env) {
     case 'test':
     case 'prod':
-
-  if (class_exists(\Composer\Autoload\ClassLoader::class)) {
-
-  $class_loader = new \Composer\Autoload\ClassLoader();
-  $class_loader->addPsr4('Drupal\\memcache\\', 'modules/contrib/memcache/src');
-
-  $class_loader->register();
-
-  $settings['container_yamls'][] = DRUPAL_ROOT . '/modules/contrib/memcache/memcache.services.yml';
+    
+      // Check for PHP Memcached libraries. 
+      $memcache_exists = class_exists('Memcache', FALSE);
+      $memcached_exists = class_exists('Memcached', FALSE);
+      if ($memcache_exists || $memcached_exists) {
```

Comment:
I think we want to keep this check.

Reply:
And maybe wrapping this is a check that `/modules/contrib/memcache/memcache.services.yml` exists as well.